source/includes/fundamentals/code-snippets/Filters.java

```diff
@@ -178,6 +194,7 @@ private void setupPaintCollection() {
     private void setupBinaryCollection(){
         List<Document> filterdata = new ArrayList<>();
         
+        // Creates a series of documents and inserts them into the "binary" collection
```

Comment:
Suggestion:
I think this should be omitted since it's setup code meant only for tech writers